spark/dl/src/main/scala/com/intel/analytics/bigdl/visualization/tensorboard/FileReader.scala

```diff
@@ -19,12 +19,14 @@ package com.intel.analytics.bigdl.visualization.tensorboard
 import java.io.{BufferedInputStream, File, FileInputStream}
 import java.nio.ByteBuffer
 
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, Path}
 import org.tensorflow.util.Event
 
 import scala.collection.mutable.ArrayBuffer
 import scala.util.matching.Regex
 
-object FileReader {
+private[bigdl] object FileReader {
```

Comment:
User should use the interface in Summary, and this FileReader should be private method.